obs-packaging/runtime/kata-runtime.spec-template

```diff
@@ -55,11 +52,10 @@ Overview
 %prep
 mkdir local
 tar -C local -xzf ../SOURCES/go%{GO_VERSION}.linux-@GO_ARCH@.tar.gz
+%autosetup -N -S git
```

Comment:
out of curiousity, any advantage of using `%autosetup` over `%setup` in this case?

Reply:
@marcov  there is not strong reason, I've seen this is a more resent version and has support for auto apply patches (thing that I disabled, but if there is any chance in the future could be used to simplify the templates).